containers/ecr-viewer/src/app/components/EcrTable.tsx

```diff
@@ -21,11 +22,13 @@ const EcrTable = async ({
   sortDirection,
   searchTerm,
   filterConditions,
+  filterDate,
 }: {
   currentPage: number;
   itemsPerPage: number;
   sortColumn: string;
   sortDirection: string;
+  filterDate: DateRangePeriod;
   searchTerm?: string;
   filterConditions?: string[];
```

Comment:
```suggestion
  searchTerm?: string;
  filterDate: DateRangePeriod;
  filterConditions?: string[];
```

super minor - just keeps the filters next to each other

Reply:
Hm I originally did that but TS said I couldn't list a required param after an optional?

Reply:
womp womp. The things we do to appease typescript 😓